src/maps/map_data_tests.cpp

```diff
@@ -6,279 +6,97 @@
 
 
 
-TEST(MapsMapdata, ExceptionThrownForMissingProvincesBmp)
+class MapsMapdata: public ::testing::Test
```

Comment:
You'll probably want to wrap all of this in a namespace {}.
From my experiments, linux will freak if it sees same-named classes, paths, or other stuff across different *Tests.cpp files.
Best to be safe.

Reply:
Good call. I actually half-expected it to fail, but it worked!